### PR TITLE
Initial import of code

### PR DESCRIPTION
**Pull request by widhalmt**
This is our initial import of working code.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+input.conf
+output.conf
```
Comment (afeefghannam89) [inline .gitignore]: I think lint will fail here **No newline at the end of file**

File: filter-50-invoked.conf [hunk 2]
```diff
@@ -0,0 +1,17 @@
+filter {
+  # remove if if really *every* message starts like this
+  if [message] =~ /Invoked with/ {
+
+    grok {
+      match => ["message", "Invoked with %{GREEDYDATA:[@metadata][ansiblekv]}"]
```
Comment (afeefghannam89) [inline filter-50-invoked.conf:6]: what does the @ do?

Comment (afeefghannam89) [reply filter-50-invoked.conf:6]: I think you store what comes after "invoked with" in **ansiblekv** then you wright this in a field name **ansible** is this right?

**Review by afeefghannam89 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (widhalmt) [reply .gitignore]: Thanks! Fixed.

File: filter-50-invoked.conf (hunk 2 above)
Comment (widhalmt) [reply filter-50-invoked.conf:6]: Right. I'm using a neat little trick: I'm writing into field `@metadata.ansiblekv`. Every field beneath `@metadata` is not written into outdputs. This way I safe an extra step for deleting a temporary field. It just get's deleted. I just pushed another change where I use this field as source for `kv`.

**Pull request merged**